File: src/main/java/tk/avabin/tdg/beans/controllers/PingRestController.java
package tk.avabin.tdg.beans.controllers;

import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

/**
 * Created by Avabin on 03.04.2017.
 */
@RestController
public class PingRestController {

    @RequestMapping("/ping")
    public String ping() {
        return "Pong!";
    }

    @RequestMapping("/pong")
    public String pong() {
        return "Hey! My job is to pong!";
    }
}

File: src/main/java/tk/avabin/tdg/beans/controllers/PingRestController.kt
package tk.avabin.tdg.beans.controllers

import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

/**
 * Created by Avabin on 03.04.2017.
 */
@RestController
class PingRestController {

    @RequestMapping("/ping")
    fun ping(): String {
        return "Pong!"
    }

    @RequestMapping("/pong")
    fun pong(): String {
        return "Hey! My job is to pong!"
    }
}
